docs/settings/storagenetwork/manual.md

```diff
@@ -0,0 +1,309 @@
+
+### Step 2: Shutdown All VMs
+
+- Stop all VMs.
+- Check all VMs are off.
+    - `kubectl get -A vmi`
+- You should not see any existing VMI.
+
+### Step 3: Shutdown All Pods with Longhorn Volumes
+
+:::caution
+
+Please stop pods by order.
+
+:::
+
+#### Stop Rancher Monitoring
+
+- Stop Rancher Monitoring.
+
+```bash
+kubectl patch -n fleet-local managedchart rancher-monitoring --type merge -p '{"spec":{"paused": true}}'
+```
+
+- Check Rancher Monitoring paused is true.
+
+```bash
+kubectl get -n fleet-local managedchart rancher-monitoring -o=jsonpath='{.spec.paused}'
+```
+
+#### Stop Grafana
+
+- Stop Grafana.
+
+```bash
+kubectl patch -n cattle-monitoring-system deployments rancher-monitoring-grafana --type merge -p '{"spec":{"replicas": 0}}'
+```
+
+- Check Deployment replicas are 0.
+
+```bash
+kubectl get -n cattle-monitoring-system deployments rancher-monitoring-grafana -o=jsonpath='{.spec.replicas}'
+```
+
+#### Stop Prometheus
+
+- Stop Prometheus.
+
+```bash
+kubectl patch -n cattle-monitoring-system prometheuses rancher-monitoring-prometheus --type merge -p '{"spec":{"replicas": 0}}'
+```
+
+- Check Prometheus replicas is 0.
+
+```bash
+kubectl get -n cattle-monitoring-system prometheuses rancher-monitoring-prometheus -o=jsonpath='{.spec.replicas}'
+```
+
+#### Stop Harvester VM Import Controller
+
+- Check VM Import Controller is existing.
+- If not, skip this step.
+
+```bash
+kubectl get -n harvester-system deployments harvester-harvester-vm-import-controller
+```
+
+- If VM Import Controller is existing, please stop Harvester VM Import Controller.
+
+```bash
+kubectl patch -n harvester-system deployments harvester-harvester-vm-import-controller --type merge -p '{"spec":{"replicas": 0}}'
+```
+
+- Check Deployment replicas are 0.
+
+```bash
+kubectl get -n harvester-system deployments harvester-harvester-vm-import-controller -o=jsonpath='{.spec.replicas}'
+```
+
+#### Stop Alertmanager
+
+- Check VM Import Controller is existing.
+- If not, skip this step.
+
+```bash
+kubectl get -n cattle-monitoring-system alertmanager rancher-monitoring-alertmanager
+```
+
+- If Alertmanager is existing, please stop Alertmanager.
+
+```bash
+kubectl patch -n cattle-monitoring-system alertmanager rancher-monitoring-alertmanager --type merge -p '{"spec":{"replicas": 0}}'
+```
+
+- Check alertmanager replicas is 0.
+
+```bash
+kubectl get -n cattle-monitoring-system alertmanager rancher-monitoring-alertmanager -o=jsonpath='{.spec.replicas}'
+```
+
+### Step 4: Check all Longhorn Volumes are detached
+
+- Please check Longhorn Volumes states are detached.
+- If there is still any attached volume, please wait for some monents and re-check it again before doing the next step.
+
+```bash
+kubectl get -A volume
+```
+
+- You could check with the following command, and should get an empty result.
+
+```bash
+kubectl get -A volume | grep attached
+```
+
+### Step 5: Setup Longhorn Storage Network Setting
+
+- Set Longhorn storage-network settings.
```

Comment:
```suggestion
- Set Longhorn storage-network.
```

Reply:
We need to check Longhorn setting as well.

Reply:
Let me check about this.

Reply:
Here we need to "set" Longhorn setting.
So "set" is the correct one.
Thanks for notice.

Reply:
Then the current sentence is OK for me~